tests/common.bash

```diff
@@ -585,3 +585,68 @@ function arch_to_kernel() {
 		*) die "unsupported architecture: ${arch}";;
 	esac
 }
+
+# Obtain a list of the files the PR changed.
+# Returns the information in format "${filter}\t${file}".
+get_pr_changed_file_details_full()
+{
+        # List of filters used to restrict the types of file changes.
+        # See git-diff-tree(1) for further info.
+        local filters=""
+
+        # Added file
+        filters+="A"
+
+        # Copied file
+        filters+="C"
+
+        # Modified file
+        filters+="M"
+
+        # Renamed file
+        filters+="R"
+
+        # Unmerged (U) and Unknown (X) files. These particular filters
+        # shouldn't be necessary but just in case...
+        filters+="UX"
```

Comment:
man pages indicate that X(Unknown) mostly indicates a bug, I would just drop it.

Reply:
@amshinde thanks, I removed this